sdk/services/exchange.ts

```diff
@@ -60,10 +61,16 @@ export default class ExchangeService {
 	private tokenList: Token[] = [];
 	private allTokensMap: any;
 	private sdk: KwentaSDK;
+	private exchangeRates: Rates = {};
 
 	constructor(sdk: KwentaSDK) {
 		this.sdk = sdk;
 		this.getOneInchTokens();
+
+		startInterval(async () => {
```

Comment:
Wondering if we should hold a reference so this could be cleared, as not a problem for us but if other clients were creating and destroying sdk instances this could lead to a memory leak. Maybe a todo is ok for now as not something we need to be concerned about with a singleton.

Reply:
I think this is a limitation of JS. The React component model makes this easy to think about in the context of effects, but we'll probably have to have a method called `clearListeners()` on the SDK class for this purpose.

Reply:
You can check this out for more context: https://stackoverflow.com/questions/29333017/ecmascript-6-class-destructor. I'm skeptical about using obscure APIs like `WeakRef` though.